packages/types/src/userSettings.ts

```diff
@@ -83,14 +83,14 @@ export type Attributes = {
 };
 
 export type Actions = {
-  delete_self: boolean;
```

Comment:
Is this intentional?

Reply:
It was not, but tests failed because of it, and fixed now!